Cargo.toml

```diff
@@ -10,6 +10,7 @@ description = "Utility to edit rolling rhino remix config"
 build = "build.rs"
 
 [dependencies]
+anyhow = "1.0.57"
```

Comment:
Could you explain what anyhow does?

Reply:
Better error messages. Plus cleaner code.

![](https://0x0.st/oTkL.png)